roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -67,7 +67,7 @@ def _print_failure_summary(self):
 # pylint: disable=protected-access
 def _format_failure(failure):
     '''Return a list of pretty-formatted lines describing a failure, including
-    relevant information about it. Line separators are not included.'''
+    relevant information about it. Lines may include line breaks.'''
```

Comment:
Not sure how much better but I think "entries".

Agree the trailing newlines should be consistently left off.